classes/Validate.php

```diff
@@ -1187,15 +1187,17 @@ public static function isLanguageFileName($file_name)
     /**
      * @param array $ids
      *
-     * @return bool return true if the array contain only unsigned int value
+     * @return bool return true if the array contain only unsigned int value and not empty
      */
     public static function isArrayWithIds($ids)
     {
-        if (count($ids)) {
-            foreach ($ids as $id) {
-                if ($id == 0 || !Validate::isUnsignedInt($id)) {
-                    return false;
-                }
+        if (count($ids) < 1) {
+            return false
```

Comment:
```suggestion
            return false;
```

unit tests are not happy :trollface:

Reply:
@kpodemski  It's Friday ! It's alcohol
Thanks man